CONTRIBUTING.md

```diff
@@ -84,3 +84,35 @@ npm --prefix aiida_workgraph/web/frontend cache clean
 ```
 
 and also clear your browsers cache or try to start new private window.
+
+
+### Building the docs
+
+We use sphinx to build the docs. You need the requirements in the extra
+`.[docs]` dependency and the `docs/requirements.txt`. We have a `docs/Makefile`
+that runs sphinx-build to build the docs.
+
+```console
+pip install .[docs]
+pip install -r docs/requirements.txt
```

Comment:
Can we copy the dependencies from the `docs` to the `requirements.txt`, so that we only need to run 
```
pip install -r docs/requirements.txt
```

Reply:
Yes I want to do it in another PR. But I actually wanted to marge them to the extras doc, as we anyway have to specify the building commands now.